pkg/features/features.go

```diff
@@ -106,7 +107,7 @@ var allFeatureGates = map[featuregate.Feature]featuregate.FeatureSpec{
 	ManagedIstio:       {Default: true, PreRelease: featuregate.Deprecated},
 	APIServerSNI:       {Default: true, PreRelease: featuregate.Deprecated},
 	SeedChange:         {Default: true, PreRelease: featuregate.Beta},
-	ReversedVPN:        {Default: true, PreRelease: featuregate.Beta},
+	ReversedVPN:        {Default: true, PreRelease: featuregate.GA, LockToDefault: true},
```

Comment:
Probably we should also lock `ManagedIstio` and `APIServerSNI` to default now given that we need them to be enabled and removed https://github.com/gardener/gardener/pull/7167/files#diff-eb33ba906d976101dfe4f69a634d451fd48618d3bfdf57fe7401aca9e03b9bd8L132-L135?

Reply:
There is also some code related to `ManagedIstio` and `APIServerSNI`. Probably, it makes sense to remove/simplify it in this PR too. What do you think?

Reply:
Yes, sounds very reasonable :)

Reply:
Why the clean up of `ManagedIstio` and `ManagedIstio` related code has to be included in this PR and cannot be done in separate PR?

Reply:
I doesn't have to be included here.
I'd also prefer a separate PR, as this one is already pretty large.
That's why I have asked.

Reply:
Yep, I would also vote to do it in a separate PR.

Reply:
Fine for me, as long as we keep them locked to default as part of this PR (otherwise, `master` will be in some strange state where you could disable these feature gates and break the entire system).

Reply:
Regarding the cleanup of the `APIServerSNI` code: I think we still need this for a while since `gardener-operator` will first use the `kubeapiserverexposure` deployer without SNI enabled to expose virtual garden `kube-apiserver`. If we now cleanup the code related to it then this will no longer be possible.
In the future, we might make `gardener-operator` also use SNI for the exposure of the virtual cluster, but this will take a few more months, I assume.
We can proceed with the cleanup of `ManagedIstio`-related code, though.